hazelcast/src/main/java/com/hazelcast/internal/serialization/InternalSerializationService.java

```diff
@@ -178,4 +180,13 @@ default <T> T readObject(ObjectDataInput in) {
 
     byte getVersion();
 
+    /**
+     * Used by tests
+     */
+    CompactStreamSerializer getCompactStreamSerializer();
+
+    /**
+     * Used by tests
+     */
+    PortableSerializer getPortableSerializer();
```

Comment:
I assume these are no longer needed

Reply:
getPortableSerializer is used in another test. So it is required. Removed the other one.

Reply:
But, we don't need to define that in this interface, right? Whatever that test is, I assume it can continue using the method defined in the implementations

Reply:
I think I can remove it as well. Let me see.